database/menuSeeding.js

```diff
@@ -0,0 +1,53 @@
+const db = require('./index.js');
+const Menu = require('./Menu.js');
+const faker = require('faker');
+var generateDishNameAndPrice = function() {
+  return {
+    item: faker.lorem.words() + ' ' + faker.lorem.words(),
+    price: '$' + faker.commerce.price().toString()}
+}
+
+var generateSectionName = function() {
+  var sections = ['Salads', 'House Specialties', 'Shareables', 'Appetizers',
+  'Starters', 'Sushi', 'Entrees', 'Dessert', 'Pasta', 'Drinks', 
+  'Hot Beverages', 'Soups', 'Tandoori', 'Seafood Entrée', 'Biryani & Rice',
+  'Vegetarian Entrée', 'Breads', 'Pizzas', 'Cheeses', 'Sides', 'Antipasti‎',
+  'Contorni‎', 'Primi‎', 'Secondi‎']
+  return faker.helpers.randomize(sections);
+}
+
+var generateMenu = function(season) {
+  return {
+    menuSeason: season,
+    sections: [
+      {
+        sectionType: generateSectionName(),
+        dishes: [
+        generateDishNameAndPrice(), generateDishNameAndPrice(), generateDishNameAndPrice(),generateDishNameAndPrice(), generateDishNameAndPrice(), generateDishNameAndPrice(),generateDishNameAndPrice(), generateDishNameAndPrice(), generateDishNameAndPrice()
```

Comment:
instead of repeating the same function call a large number of times (it's a pain to update and is prone to errors), consider something like:
dishes: Array(9).fill(null).map(() => generateDishNameAndPrice()),